packages/ove-app-audio/src/client/common/audio.js

```diff
@@ -33,9 +33,8 @@ initCommon = function () {
             setTimeout(function () {
                 switch (op.name) {
                     case Constants.Operation.PLAY:
-                        log.info('Starting audio playback ' + (op.loop ? 'with' : 'without') + ' loop'
-                                                            + (op.volume === undefined ? 'without' : 'with '+op.volume) + ' volume ');
-                        context.player.play(op.loop,op.volume);
+                        log.info('Starting audio playback ' + (op.loop ? 'with' : 'without') + ' loop' + (op.volume === undefined ? 'without' : 'with ' + op.volume) + ' volume ');
```

Comment:
This log line would not look right when printed out. Suggest that it be fixed as below.

```suggestion
                        log.info('Starting audio playback ' + (op.loop ? 'with' : 'without') + ' loop and ' + (op.volume === undefined ? 'without' : 'with ' + op.volume) + ' volume ');
```